lib/Sema/CSSimplify.cpp

```diff
@@ -4774,6 +4775,26 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyMemberConstraint(
                                       flags, locatorB);
     };
 
+    // Check if any property wrappers on the base of the member lookup have
+    // mactching members that we can fall back to.
+    if (auto dotExpr =
+            dyn_cast_or_null<UnresolvedDotExpr>(locator->getAnchor())) {
+      auto baseExpr = dotExpr->getBase();
+      auto resolvedOverload = findSelectedOverloadFor(baseExpr);
+
+      auto wrappedProperty = getPropertyWrapperInformation(resolvedOverload);
+      if (wrappedProperty) {
```

Comment:
Nit: could be simplified to `if (auto info = getPropertyWrapperInformation(resolvedOverload))` since Optional overloads bool() operator